src/004/p004.exs

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env elixir
+defmodule Problem004 do
+  defp reverse_number(x) do
+  	x

```

Comment:
This should be indented to match the pipes.

Reply:
Oh, my bad.. just noticed it's not correctly indented!